CMR/python/cmr/search/granule.py

```diff
@@ -0,0 +1,119 @@
+# NASA EO-Metadata-Tools Python interface for the Common Metadata Repository (CMR)
+#
+#     https://cmr.earthdata.nasa.gov/search/site/docs/search/api.html
+#
+# Copyright (c) 2020 United States Government as represented by the Administrator
+# of the National Aeronautics and Space Administration. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software distributed
+# under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR
+# CONDITIONS OF ANY KIND, either express or implied. See the License for the
+# specific language governing permissions and limitations under the License.
+
+"""
+A Library for building and requesting CMR granule searches
+date: 2020-11-23
+since: 0.0
+
+A simple search interface for CMR. Use the search() function to perform searches.
+This function can handle any query parameter which is supported by the CMR.
+
+    search()
+        query - a dictionary of CMR parameters
+        filters - a list of result filter lambdas
+        limit - int limiting the number of records returned
+        config - configurations
+
+More information can be found at:
+https://cmr.earthdata.nasa.gov/search/site/docs/search/api.html
+"""
+
+import cmr.search.common as scom
+
+# ******************************************************************************
+# filter function lambdas
+
+def all_fields(item):
+    """ Pass through, do no filtering - is this needed outside of testing?"""
```

Comment:
Turn the question into an imperative instead.

Reply:
resulted in adding some files with similar comments, marking an resolved.